AdaptiveCards/templating/index.md

```diff
@@ -12,7 +12,16 @@ We're excited to share a preview of new tools that will help you **create**, **r
 
 > [!IMPORTANT] 
 > 
-> These features are **in preview and subject to change**. Your feedback is not only welcome, but  critical to ensure we deliver the features **you** need.
+> **Breaking changes** in the **May 2020 Release Candidate**
+>
+> We've been hard at work getting templating released, and we're finally in the home stretch! We had to make some minor breaking changes as we close on the release.
```

Comment:
See [recommendations[(https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=28361) against using first person plural and instead trying to work around it if you can, where possible. 
```suggestion
> The new templating release candidate package includes some minor breaking changes that you should be aware of if you've been using the older packages. Troubleshoot common migration or usage issues in the [Troubleshooting section below](Add link to the section here).
```